app/helpers/spots_helper.rb

```diff
@@ -10,4 +10,12 @@ def display_day(date)
       date.strftime("dd.mm.")
     end
   end
+
+  def render_with_hashtags(text)
+    sanitize(text.gsub(/(?:#(\w+))/) {hashtag_link($1)})
```

Comment:
Is `sanitize` by default gonna leave the link there? (Just asking, cause I'm too lazy to check.)